components/collection/drop/GenerativePreview.vue

```diff
@@ -123,38 +126,22 @@ const mintedPercent = computed(() => {
   }
   return Math.round((mintedCount.value / maxCount.value) * 100)
 })
-const entropyRange = computed<[number, number]>(() => [
-  STEP * mintedAmountForCurrentUser.value,
-  STEP * (mintedAmountForCurrentUser.value + 1),
-])
```

Comment:
i like that you moved it out
but this is a simple util function and does not belong in a composable

```typescript
const entropyRange(num: number, step = 64): [number, number]
```